registry.d.ts

```diff
@@ -0,0 +1,10 @@
+declare class registry {
```

Comment:
Move type definition to proper location.

Reply:
Done.